support/python/tests/test_events.py

```diff
@@ -0,0 +1,16 @@
+from nebula_sdk.client import new_session
+from nebula_sdk.events import Events
+from requests_mock import Adapter
+
+
+class TestEvents:
+
+    def test_emit(self, requests_mock: Adapter) -> None:
+        requests_mock.register_uri(
+            'POST', 'http+api://api/events',
+            text='OK',
+            request_headers={'content-type': 'application/json'},
+            additional_matcher=lambda request:
+                request.json() == {'data': {'foo': 'bar'}},
+        )
+        Events(new_session(api_url='http://api')).emit({'foo': 'bar'})
```

Comment:
this test begs the question...what behavior do we expect when this fails? and how do we expect our system/our users to handle that? it's an edge case, but should be considered as part of the "logging" topic.

Reply:
Most (all?) upstream providers retry webhooks for this reason, although we should probably make sure our internal retry logic for these requests is sound too.